### DISCUSSION TITLE
Hi all, I'm Kalhara, a Computer Engineering Undergraduate.

### DISCUSSION THREAD

**Discussion by KalharaBatangala** [community/community#138912] [New to GitHub]

**Comment by ngchthien** [comment]
Hi Kalhara, it's great to meet you! It's awesome that you're pursuing Computer Engineering and already diving into Android development. That's a really cool combination.
To give you useful feedback, could you tell me a bit more about what you'd like feedback on specifically? For example:
Are you working on a particular Android project right now? If so, what kind of app is it? What are some of the challenges you're facing?
What aspects of Android development are you most interested in? (UI/UX, backend, specific libraries or frameworks)
What are your goals as an Android developer? (personal projects, internships, career aspirations)
The more information you share, the more relevant and helpful feedback I can provide. Don't hesitate to be specific!
Looking forward to hearing more about your journey as an Android developer!
